### PR TITLE
Add usage information

### PR DESCRIPTION
**Pull request by Karnaukhov-kh**
I just got a question about how & what to import.

### CODE REVIEW

File: libs/template/README.md [hunk 1]
```diff
@@ -53,6 +53,21 @@ are several techniques that need to be known and considered.
 or  
 `yarn add @rx-angular/template`
 
+## Usage
+
+Import `TemplateModule` to your `AppModule`.
```
Comment (Karnaukhov-kh) [inline libs/template/README.md:58]:
```suggestion
Import `TemplateModule` to your Module.
```

Comment (Karnaukhov-kh) [reply libs/template/README.md:58]: Importing in AppModule does not work.  "Push pipe can not be found"

File: libs/template/README.md [hunk 2]
```diff
@@ -53,6 +53,21 @@ are several techniques that need to be known and considered.
...
+
+```typescript
+import { TemplateModule } from '@rx-angular/template';
+
+@NgModule({
+  declarations: [...],
+  imports: [TemplateModule],
+})
+export class AppModule {}
```
Comment (Karnaukhov-kh) [inline libs/template/README.md:67]:
```suggestion
export class MyModule {}
```

File: libs/template/README.md [hunk 3]
```diff
@@ -53,6 +53,21 @@ are several techniques that need to be known and considered.
...
+```
+
+
```
Comment (Karnaukhov-kh) [inline libs/template/README.md:70]:
```suggestion
## Usage

Import `TemplateModule` to your Module.

```typescript

import { TemplateModule } from '@rx-angular/template';

@NgModule({
  declarations: [...],
  imports: [TemplateModule],
})
export class MyModule {}
```

Alternatively, you can import `LetModule` or `PushModule` individually.

```typescript
import { LetModule, PushModule } from '@rx-angular/template';

@NgModule({
  declarations: [...],
  imports: [LetModule, PushModule],
})
export class MyModule {}
```

```

Comment (Karnaukhov-kh) [reply libs/template/README.md:70]: @hoebbelsB smth like this?

File: libs/template/README.md [hunk 4]
```diff
@@ -53,6 +53,19 @@ are several techniques that need to be known and considered.
...
+```typescript
+
+import { TemplateModule } from '@rx-angular/template';
+
+@NgModule({
+  declarations: [...],
+  imports: [TemplateModule],
+})
+export class MyModule {}
```
Comment (Karnaukhov-kh) [inline libs/template/README.md:68]:
```suggestion
## Usage

Import `TemplateModule` to your Module. Alternatively, you can import LetModule or PushModule individually.

```typescript
// Importing full TemplateMdoule.
import { TemplateModule } from '@rx-angular/template';

@NgModule({
  declarations: [...],
  imports: [TemplateModule],
})
export class MyModule {}

// Importing LetModule & PushModule individually.
import { LetModule, PushModule } from '@rx-angular/template';

@NgModule({
  declarations: [...],
  imports: [LetModule, PushModule],
})
export class MyModule {}
```

File: libs/template/README.md [hunk 5]
```diff
@@ -53,6 +53,28 @@ are several techniques that need to be known and considered.
 or  
 `yarn add @rx-angular/template`
 
+## Usage
+
+Import `TemplateModule` to your Module. Alternatively, you can import LetModule or PushModule individually.
+
+```typescript
+// Importing full TemplateMdoule.
```
Comment (hoebbelsB) [inline libs/template/README.md:61]:
```suggestion
// Importing TemplateModule (includes push, *rxLet, unpatch and viewport-prio)
```

File: libs/template/README.md [hunk 6]
```diff
@@ -53,6 +53,28 @@ are several techniques that need to be known and considered.
 or  
 `yarn add @rx-angular/template`
 
+## Usage
+
+Import `TemplateModule` to your Module. Alternatively, you can import LetModule or PushModule individually.
```
Comment (hoebbelsB) [inline libs/template/README.md:58]:
```suggestion
Import `TemplateModule` to your Module. Alternatively, you can import each feature module individually.
```

File: libs/template/README.md [hunk 7]
```diff
@@ -53,6 +53,28 @@ are several techniques that need to be known and considered.
...
+import { TemplateModule } from '@rx-angular/template';
+
+@NgModule({
+  declarations: [...],
+  imports: [TemplateModule],
+})
+export class MyModule {}
+
+// Importing LetModule & PushModule individually.
+import { LetModule, PushModule } from '@rx-angular/template';
```
Comment (hoebbelsB) [inline libs/template/README.md:71]:
```suggestion
import { LetModule, PushModule, ViewportModule, UnpatchEventsModule} from '@rx-angular/template';
```

File: libs/template/README.md [hunk 8]
```diff
@@ -53,6 +53,28 @@ are several techniques that need to be known and considered.
...
+
+@NgModule({
+  declarations: [...],
+  imports: [LetModule, PushModule],
```
Comment (hoebbelsB) [inline libs/template/README.md:75]:
```suggestion
  imports: [LetModule, PushModule, ViewportModule, UnpatchEventsModule],
```

**Pull request merged**